css/highcharts.css

```diff
@@ -1249,6 +1249,7 @@ text.highcharts-drilldown-data-label,
     margin: 0;
     display: block;
     position: absolute;
+    pointer-events: "none";
```

Comment:
This looks wrong, CSS doesn't support quoted property values. Can you have another check?